src/app/actions.js

```diff
@@ -2,7 +2,8 @@
 export const actionsList = {
   storeBakery: "STORE_BAKERY",
   storeVisitURL: "STORE_VISIT_URL",
-  updateControllerConnection: "UPDATE_CONTROLLER_CONNECTION"
+  updateControllerConnection: "UPDATE_CONTROLLER_CONNECTION",
+  LogOut: "LOG_OUT"
```

Comment:
Lets use something more descriptive `clearUserData`. This will allow us to compose actions in the future if needed.